.env.example

```
@@ -8,6 +8,12 @@ DATABASE_URL="postgresql://user:password@localhost:5432/dappdrop"
 NEXT_PUBLIC_CAMPAIGN_FACTORY_CONTRACT="0x..."
 NEXT_PUBLIC_SEPOLIA_RPC_URL="https://rpc.sepolia.org"
 
+# === THE GRAPH (optional but recommended for performance) ===
+# Deploy the subgraph in /subgraph to Subgraph Studio, then paste the query URL here.
+# When set, campaign list and participant queries use the subgraph instead of N+1 RPC calls.
+# Format: https://api.studio.thegraph.com/query/<id>/dappdrop/version/latest
+NEXT_PUBLIC_GRAPH_API_URL=""
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)